pkg/shell/nav.tsx

```diff
@@ -27,8 +27,9 @@ import { SearchInput } from "@patternfly/react-core/dist/esm/components/SearchIn
 import { Tooltip, TooltipPosition } from "@patternfly/react-core/dist/esm/components/Tooltip/index.js";
 import { ContainerNodeIcon, ExclamationCircleIcon, ExclamationTriangleIcon, InfoCircleIcon } from '@patternfly/react-icons';
 
+import { Status } from "notifications";
```

Comment:
This is a case where `import type` is preferable, as notifications.tsx isn't otherwise used in the shell. However, we recently talked about that, and

    NODE_ENV=development ./build.js shell

indeed confirms that pkg/lib/notifications.tsx does not land in the dist/shell/shell.js shell bundle. So tree shaking does work for pure .js/.ts, just not for anything with CSS.

It may still be a good habit to learn, to not unnecessarily blow up our bundles.

Reply:
Yes, good point! I didn't think of that.